docs/ossm/ossm2-migration/cluster-wide/README.md

```diff
@@ -0,0 +1,384 @@
+1. Apply correct labels to the data plane namespace
+
+    Here we're adding two and removing one label:
+
+    1. The `istio.io/rev=ossm-3-v1-24-1` label which ensures that any new pods that get created in that namespace will connect to the 3.0 proxy. In our example, the 3.0 revision is named `ossm-3-v1-24-1`
+    1. The `maistra.io/ignore-namespace: "true"` label which will disable sidecar injection for 2.6 proxies in the namespace. This ensures that 2.6 will stop injecting proxies in this namespace and any new proxies will be injected by the 3.0 control plane.
+    1. It's necessary to temporarily remove the `istio-injection=enabled` as it would prevent the proxy injection by 3.0 control plane.
+
+    ```sh
+    oc label ns bookinfo istio.io/rev=ossm-3-v1-24-1 maistra.io/ignore-namespace="true" istio-injection- --overwrite=true
+    ```
+
+1. Migrate workloads
+
+    You can now restart the workloads so that the new pods will be injected with the 3.0 proxy.
+
+    This can be done all at once:
+
+    ```sh
+    oc rollout restart deployments -n bookinfo
+    ```
+    or individually:
+    ```sh
+    oc rollout restart deployments productpage-v1 -n bookinfo
+    ```
+1. Validation of the workloads can be done the same way as in the previous procedures
+
+> [!CAUTION]
+> Before proceeding, it's necessary to finish migration of all remaining namespaces.
+
+##### Create a default revision tag and re-label namespaces
+1. Prepare a default revision tag yaml named `rev-tag.yaml`:
+    ```yaml
+    apiVersion: sailoperator.io/v1alpha1
+    kind: IstioRevisionTag
+    metadata:
+      name: default
+    spec:
+      targetRef:
+        kind: IstioRevision
+        name: ossm-3-v1-24-1
+    ```
+1. Apply the `rev-tag.yaml`:
+    ```sh
+    oc apply -f rev-tag.yaml
+    ```
+1. Verify `IstioRevisionTag` status:
+    ```sh
+    oc get istiorevisiontags
+    NAME      STATUS                    IN USE   REVISION        AGE
+    default   NotReferencedByAnything   False    ossm-3-v1-24-1  18s
+    ```
+1. Add `istio-injection=enabled` and remove `istio.io/rev` label:
+    ```sh
+    oc label ns bookinfo istio-injection=enabled istio.io/rev-
+    ```
+1. Restart the workloads:
+    ```sh
+    oc rollout restart deployments -n bookinfo
+    ```
+1. Verify the `IstioRevisionTag` is in use:
+    ```sh
+    oc get istiorevisiontags
+    NAME      STATUS    IN USE   REVISION        AGE
+    default   Healthy   True     ossm-3-v1-24-1  28s
+    ```
+1. Validation of the workloads can be done the same way as in the previous procedures
+1. Repeat steps 4. and 5. for other namespaces
+
+#### Simple migration of 2.6 installation with istio-injection=enabled label
+It is recommended to use one of the procedures above in production environments.
+In this procedure it's expected that all 2.6 data plane namespaces have `istio-injection=enabled` label.
+> [!CAUTION]
+> This procedure may cause a traffic disruption for workloads which are restarted at unexpected time.
```

Comment:
Can you expand on the scenario(s) that may cause traffic disruption here?

Reply:
It's explained in the note bellow when applying the Istio resource yaml. I will add more info here too.